share/graphql/query.py

```diff
@@ -41,7 +41,7 @@ def resolve_me(self, args, context, info):
     @graphene.resolve_only_args
     def resolve_sources(self, limit=25, offset=0):
         # TODO Long title will need an index
-        return models.ShareUser.objects.exclude(long_title='').exclude(favicon='').order_by('long_title')[offset:offset + limit]
+        return models.ShareUser.objects.exclude(long_title='').exclude(icon='').order_by('long_title')[offset:offset + limit]
```

Comment:
Ah sorry negative query confusion. I meant ~(P OR Q) which is the same as ~P AND ~Q. Did not know how to see the query though, that's neat.

I didn't look into that method though...seems like the whole thing needs to change if it's being used?